AndroidOnChromeOSDragAndDropDemo/settings.gradle

```diff
@@ -15,6 +15,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://androidx.dev/snapshots/latest/artifacts/repository' }
```

Comment:
Thanks, Ben! Let's remove this and update the dependency to target the alpha release when it goes live on the 15th.

Reply:
I've made this change in #305.